src/components/Filter/components/Options/Options.tsx

```diff
@@ -33,16 +35,37 @@ const Options: React.FC<Props> = ({
 
   return items.length ? (
     <List
-      data={items.filter((option) => option.value !== defaultValue.value)}
-      rowSize={'small'}
-      defaultOption={defaultOption}
-      selectedItem={selectedItem}
-      isSearchable={isSearchable}
-      handleOptionClick={(option: FilterOption) => onSelect(option)}
+      label={'filter-options'}
+      selectedKeys={selectedItem ? [selectedItem.value] : []}
+      disabledKeys={items.filter((o) => o.isDisabled).map((o) => o.value)}
+      onSelectionChange={(keys) => {
```

Comment:
same here and for all anonymous functions of this PR

Reply:
solved at [`3b16bc4` (#655)](https://github.com/Orfium/orfium-ictinus/pull/655/commits/3b16bc4ae94f7c2e343aaf69ff08fb13226e39d5)